src/SerializerFactory.php

```diff
@@ -94,15 +111,25 @@ private function shouldSerializeReferenceSnaksWithHash() {
 	}
 
 	/**
-	 * Returns a Serializer that can serialize Item and Property objects.
-	 *
-	 * @return Serializer
+	 * @throws IllegalValueException
+	 * @return Serializer A dispatching serializer that can serialize whatever the entity
```

Comment:
DispatchingSerializer is not private and we could expose it here. But there is not much value in this. The only difference is that this makes the `addSerializer` method visible. This should not be needed.

I'm always putting `@throws` before `@return`, because it reflects the actual control flow: code can either throw exceptions for different reasons, or return something in the end.